environment-dev.yml

```diff
@@ -7,15 +7,15 @@ dependencies:
   - cxx-compiler
   # Host dependencies
   - nlohmann_json
-  - xtl
   - cppzmq
```

Comment:
I think this one can be removed too.

Reply:
which one? xtl **is beeing removed**

Reply:
I think Johan means that cppzmq can be removed too